game_event/test_game_event_views.py

```diff
@@ -1,28 +1,22 @@
 import pytest
 from django.urls import reverse
-from .models import GameEvent, Court
-from decimal import Decimal
+from .models import GameEvent
 from .views import game_events
 from django.utils import timezone
 
 
 @pytest.mark.django_db
 class TestGameEventServerResponses:
-    @pytest.fixture
-    def saved_court(self):
-        court = Court.objects.create(x=Decimal('11'), y=Decimal('22'),
-                                     city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
-        return court
 
     @pytest.fixture
-    def saved_game_event(self, saved_court):
+    def saved_game_event(self, court):
```

Comment:
Can we also add this `saved_game_event` to the `conftest.py` file?

Reply:
I want to do it next, I'm waiting for this PR to be merged. Daniel wanted the change all of the fixtures in the same PR, the mentors requested to make the PR's smaller. so for your question I'm on it (: